ClimateMonitorV2/views/index.pug

```diff
@@ -119,11 +119,14 @@ block content
                             a(class="carousel-control-next" href="#pythongraphslideshow" role="button" data-slide="next")
                                 span(class="carousel-control-next-icon" aria-hidden="true" style="background-color: blue")
                                 span(class="sr-only") Next
-    
+
      script(type='text/javascript' src='/javascripts/config.js')
      script(type='text/javascript', src='/javascripts/new_mapping.js')
   script(src="https://d3js.org/d3.v5.min.js")
   script(src="https://cdn.jsdelivr.net/npm/litepicker/dist/js/main.js")
 
-  footer
-    a(href="https://www.gov.uk/government/publications/health-matters-air-pollution/health-matters-air-pollution?fbclid=IwAR1WCj4WuVHtGuDehhsQZ682MUpF4MIdM2AL8i4Z6UvyjnIkOiePaYjrQks") Information and images from Gov.UK
+  footer(id = "sticky-footer" class="py-4 bg-light text-white-50")
+    div(class="container text-center")
+        a(href="https://urbanflows.ac.uk/")
+            img(class="img-fluid", src="images/urbanflows.png", href="https://urbanflows.ac.uk/") 
+        //a(href="https://www.gov.uk/government/publications/health-matters-air-pollution/health-matters-air-pollution?fbclid=IwAR1WCj4WuVHtGuDehhsQZ682MUpF4MIdM2AL8i4Z6UvyjnIkOiePaYjrQks") Information and images from Gov.UK
```

Comment:
Can we remove the comment? It's on the about page so we shouldn't really need it again here

Reply:
Good point, removing it